src/backend/distributed/planner/distributed_planner.c

```diff
@@ -835,16 +838,9 @@ GetDistributedPlan(CustomScan *customScan)
 	node = (Node *) linitial(customScan->custom_private);
 	Assert(CitusIsA(node, DistributedPlan));
 
-	node = CheckNodeCopyAndSerialization(node);
+	CheckNodeCopyAndSerialization(node);
```

Comment:
I changed this line to also not return a copied plan in the regression tests. This change helped me catch a bug in my initial implementation.

Reply:
I am okay with that.